tests/cases/fourslash/findAllRefsImportEqualsJsonFile.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference path='fourslash.ts' />
+
+// @allowJs: true
+// @checkJs: true
+// @resolveJsonModule: true
+
+// @Filename: /a.ts
+////import [|{| "isWriteAccess": true, "isDefinition": true |}j|] = require("[|./j.json|]");
+////[|j|];
+
+// @Filename: /b.js
+////const [|{| "isWriteAccess": true, "isDefinition": true |}j|] = require("[|./j.json|]");
+////[|j|];
+
+// @Filename: /j.json
+////[|{ "x": 0 }|]
+
+verify.noErrors();
+
+const [r0, r1, r2, r3, r4, r5, r6] = test.ranges();
+verify.singleReferenceGroup('import j = require("./j.json")', [r0, r2]);
+verify.referenceGroups([r1, r4], [{ definition: 'module "/j"', ranges: [r1, r4, r6] }]);
+verify.singleReferenceGroup('const j: {\n    "x": number;\n}', [r3, r5]);
+verify.referenceGroups(r6, undefined);
```

Comment:
Should this trigger on the open brace of the json file? I don't want to just trigger it everywhere since people may be looking for references to a particular property instead.
But vscode at least doesn't use TypeScript to provide services in json files, so this wouldn't work anyway ... we can still get references when starting from a TS file though.